reseq_ros2/config/reseq_mk1_can.yaml

```diff
@@ -11,3 +12,23 @@ modules:
   - address: 0x13
     hasJoint: true
     hasEndEffector: false
+geometric_consts:
+  a: 0.18                    # [m] Distance between the center of every module and the previous yaw joint
+  b: 0.18                    # [m] Distance between the center of every module and the following yaw joint
```

Comment:
These are not the measured values for the mk1: you can use these [here](https://github.com/Team-Isaac-Polito/reseq_ros2/blob/4a9232a473ddb054e590d0b1f5a4474bd40532ee/reseq_ros2/config/config_mk1.yaml#L15C1-L16C95)